### PR TITLE
Change admin split command to only take the split key

### PR DESCRIPTION
**Pull request by mberhault**
The "key" arg was only really needed to find the range. Beyond that, people manually splitting ranges will usually have a specific split key in mind and won't want automatic split determination.

### CODE REVIEW

**Comment by spencerkimball** [comment]
LGTM

File: server/cli/range.go [hunk 1]
```diff
@@ -78,33 +78,26 @@ func runLsRanges(cmd *cobra.Command, args []string) {
...
 // A splitRangeCmd command splits a range.
 var splitRangeCmd = &cobra.Command{
-	Use:   "split [options] <key> [<split-key>]",
+	Use:   "split [options] <key>",
 	Short: "splits a range",
 	Long: `
-Splits the range containing <key>. If <split-key> is not specified a
-key to split the range approximately in half will be automatically
-chosen.
+Splits the range containing at <key>.
```
Comment (petermattis) [inline server/cli/range.go]: `Splits the range containing <key> at <key>.`

**Comment by petermattis** [comment]
LGTM

File: server/cli/range.go (hunk 1 above)
Comment (MBerhault) [inline server/cli/range.go]: Done.

**Pull request merged**